docs/UpgradingToJest28.md

```diff
@@ -182,3 +182,7 @@ import {jest} from '@jest/globals';
     .mockResolvedValue('default')
     .mockResolvedValueOnce('first call');
 ```
+
+## Babel config
```

Comment:
Perhaps could go up? Or under `Configuration`? The idea was to keep alphabetical order, hoping that this could help the reader. But not necessary it works as I was thinking (;

Reply:
fair, moved it 👍